src/deadline/job_attachments/incremental_downloads/orchestrator.py

```diff
@@ -0,0 +1,162 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I expect the overall structure of this command to be better if there is no concept like "orchestrator" in the design, so I recommend you remove this file.

As a better alternative, think about the lower level operations the command needs to run, and structure them into a callable library form. Those operations should be self-contained and isolated from the overall command, they should look like useful library calls that other commands could use as well.